plugins/inputs/t128_tank/t128_tank.go

```diff
@@ -118,20 +126,29 @@ func (plugin *T128Tank) Stop() {
 	plugin.mainWG.Wait()
 }
 
-func extractTopicType(message string, pattern string) string {
-	regex := regexp.MustCompile(pattern)
-	match := regex.FindStringSubmatch(message)
-	if len(match) > 1 {
-		return match[1]
-	}
-	return ""
-}
-
 func (plugin *T128Tank) checkConfig() error {
 	if plugin.Topic == "" {
 		return fmt.Errorf("topic is a required configuration field")
 	}
 
+	if plugin.IndexFile == "" {
```

Comment:
Sorry, one more request that I didn't make before. This will be used for the new analytics sync work. In that, we'll want to reuse some existing indices to prevent a bunch of unnecessary work on upgrade. Can you make the index reading code only consider the first line? Then existing SSR index files will be usable with this input. Here's an example of the existing index files.
```
[root@t263-dut1 centos]# cat /var/lib/128technology/streams/local-persistent_events.index
888
1697659128000
```